app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java

```diff
@@ -208,6 +213,7 @@ public void onPairingButtonClicked(final View view) {
             final BluetoothAdapter bluetoothAdapter = BluetoothAdapter.getDefaultAdapter();
             if (bluetoothAdapter.isEnabled()) {
                 intent.setComponent(new ComponentName(this, BluetoothConnectionActivity.class));
+                intent.addFlags(Intent.FLAG_ACTIVITY_NO_ANIMATION);
```

Comment:
Using "singleinstance" for the BluetoothConnectionActivity gives a very ugly and large animation. This removes it.